### PR TITLE
mds: minor balancer fixes

### PR DESCRIPTION
**Pull request by ukernel**

### CODE REVIEW

File: src/mds/MDBalancer.cc [hunk 1]
```diff
@@ -1134,7 +1134,9 @@ void MDBalancer::maybe_fragment(CDir *dir, bool hot)
 {
   // split/merge
   if (bal_fragment_dirs && bal_fragment_interval > 0 &&
-      dir->is_auth() && !dir->inode->is_base()) {  // not root/base (for now at least)
+      dir->is_auth() &&
+      !dir->inode->is_base() &&  // not root/mdsdir (for now at least)
+      !dir->inode->is_stray()) { // not straydir
```
Comment (batrick) [inline src/mds/MDBalancer.cc]: Why? What issue is this fixing? Please make a tracker ticket.

File: src/mds/MDBalancer.cc [hunk 2]
```diff
@@ -859,10 +859,10 @@ void MDBalancer::try_rebalance(balance_state_t& state)
     mds_rank_t target = it.first;
     double amount = it.second;
 
-    if (amount / target_load < .2)
-      continue;
     if (amount < MIN_OFFLOAD)
       continue;
+    if (amount * 10 * state.targets.size() < target_load)
+      continue;
```
Comment (batrick) [inline src/mds/MDBalancer.cc]: Needs a tracker ticket. Separate PR too please for backport.

File: src/mds/MDBalancer.cc (hunk 1 above)
Comment (ukernel) [reply src/mds/MDBalancer.cc]: The issue is trivial. Above code queue straydir (can not be fragmented) for split.  Just waste few cpu cycles

Comment (batrick) [reply src/mds/MDBalancer.cc]: Okay, I'm okay with this one not having a ticket but the other commit does.

**Pull request merged**